### PR TITLE
Adds additional aerialway presets, fixes #3733

### LINKED ISSUE #3733: Missing important aerialway presets
aerialway=drag_lift
aerialway=goods
aerialway=mixed_lift

### PR DESCRIPTION
**Pull request by ajithranka**
cc: @bhousel @kepta

### CODE REVIEW

**Comment by bhousel** [comment]
Thanks @ajithranka!

File: data/presets/presets.json [hunk 1]
```diff
@@ -223,6 +236,19 @@
...
+                "line"
+            ],
+            "fields": [
+                "aerialway/capacity",
+                "aerialway/duration"
+            ],
+            "tags": {
+                "aerialway": "goods"
+            },
+            "name": "Goods"
```
Comment (willemarcel) [inline data/presets/presets.json]: I think 'Goods' does not describes properly this tag. What do you think about to change it to 'Goods Aerialway' or 'Cable Lift for Goods'

Comment (bhousel) [inline data/presets/presets.json]: Sure that's a good idea.  Either of those sound fine.. 🚠

Comment (bhousel) [inline data/presets/presets.json]: I just renamed it to "Goods Aerialway"

**Pull request merged**